strato

```diff
@@ -124,6 +124,24 @@ function fetchLogs {
   python fetchlogs ${withdb_flag}
 }
 
+function setPassword {
+  while [ -z $PASSWORD ]; do
+    echo
+    echo -n Please enter a password:
+    read -s PASSWORDA
+    echo
+    echo -n Please re-enter the password:
+    read -s PASSWORDB
+    if [ "${PASSWORDA}" == "${PASSWORDB}" ]; then
+      PASSWORD=$PASSWORDA
+    else
+      echo
+      echo -n Passwords did not match. Please try again.
+    fi
+  done
+  (docker exec -i strato_vault-wrapper_1 curl --silent -H "Content-Type: application/json" -d @- localhost:8000/strato/v2.3/password <<< \"$PASSWORD\") > /dev/null
```

Comment:
Vault-wrapper does not keep the request body for post requests in the logs, does it?
I want to avoid having the password in the logs (even with debug log level)

Reply:
It's fine - this is what I see in vault-wrapper logs:
```
[2019-06-28 09:07:48.334662107 UTC]  INFO | ThreadId 82    |                                     | SELECT
"salt1_1" as "result1_2",
"nonce2_1" as "result2_2",
"enc_msg3_1" as "result3_2"
FROM (SELECT
      *
      FROM (SELECT
            "id" as "id0_1",
            "salt" as "salt1_1",
            "nonce" as "nonce2_1",
            "enc_msg" as "enc_msg3_1"
            FROM "message" as "T1") as "T1"
      WHERE (("id0_1") = 1)) as "T1"
[2019-06-28 09:07:48.338709516 UTC]  INFO | ThreadId 82    |                                     | Updating the database
127.0.0.1 - - [28/Jun/2019:09:07:48 +0000] "POST /strato/v2.3/password HTTP/1.1" 200 - "" "curl/7.58.0"
```
postgres logs don't have the values as well.

Hiding values in debug mode is probably the overly, since prod nodes never have debug log level